### PR TITLE
fix: allow live preview iframe src (#80)

### PR DESCRIPTION
**Pull request by YvesRijckaert**
Allow live preview iframe src and upgrade node version

### CODE REVIEW

**Review by chrishelgert — APPROVED** [review]

File: config/headers.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const securityHeaders = [
   },
   {
     key: 'X-Frame-Options',
-    value: 'SAMEORIGIN',
+    value: 'SAMEORIGIN https://app.contentful.com',
   },
```
Comment (cf-remylenoir) [inline config/headers.js:13]: Are you also gonna update the headers according to https://github.com/contentful/template-marketing-webapp-nextjs/pull/195/files#diff-29b59c32b97c6884e94911617077b4059b076906b21417f7d61419c9deb5270e ?

File: .nvmrc [hunk 2]
```diff
@@ -1 +1 @@
-v18.13.0
+v18.15
```
Comment (cf-remylenoir) [inline .nvmrc:1]:
```suggestion
v18.15.0
```

**Review by cf-remylenoir — APPROVED** [review]
LGTM but left a comment (disabled the auto-merge for that reason)

File: config/headers.js (hunk 1 above)
Comment (YvesRijckaert) [reply config/headers.js:13]: yes thats part of a different ticket in our sprint, this was just to unblock customers atm

Comment (cf-remylenoir) [reply config/headers.js:13]: Gotcha, thanks!

**Pull request merged**